stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -26,9 +20,10 @@ const SecuritiesTable = () => {
       .then(securities => {
         setIsLoading(false);
         setSecuritiesData(securities);
-        setErrorMessage(null)
+        setErrorMessage(null);
       })
-      .catch(() => {
+      .catch((e) => {
+        console.log(e);
```

Comment:
You don't need this `console.log` any more. It was just to illustrate what would come through in the event of an error.

Reply:
Yep understood